src/moonstone-samples/src/ToggleButtonSample.js

```diff
@@ -14,7 +14,7 @@ var
 module.exports = kind({
 	name: 'moon.sample.ToggleButtonSample',
 	kind: FittableRows,
-	classes: 'moon enyo-unselectable enyo-fit',
+	classes: 'moon enyo-unselectable enyo-fit moon-toggle-button-sample',
```

Comment:
I assume this is just added for completeness, as it otherwise does not have any CSS associated with it?